### ISSUE TITLE
1/12/2023: Marc Berman

### ISSUE THREAD

**Issue by GabeNicholson** [uchicago-computation-workshop/Winter2023#1]
Comment below with a well-developed question or comment about the reading for this week's workshop.

If you would really like to ask your question in person, please place two exclamation points before your question to signal that you really want to ask it.

Please post your question by Tuesday of the coming week, at 11:59 PM. We will also ask you all to upvote questions that you think were particularly good. There may be prizes for top question-askers.

**Comment by sdbaier** [comment]
I am afraid that I do not fully grasp the assumptions and logic of the model on homophily and heterophobia, and/or parts of it may partly proceed in circular fashion.

If my understanding of the _Effects of Racial Segregation on Economic Productivity in U.S. Cities_ is correct, then the coefficients for homophily and heterophobia capture both structural factors and (potential) agentic choices that result in network segregation (i.e., outcome homophily and heterophobia, as described on p. 5). Individuals do not interact with each other equally likely, but belong to different groups, each with their own group-specific interaction rate. Individuals in each group are modelled as interacting preferentially with others of the same group, and with a lower probability with other groups. In case they lose contact with other groups, they may increase their rate of intra-group interactions (see p. 4). Put differently, the base assumption is some degree of homophily (preferential interaction within-group, be it structural and/or agentic) and heterophobia (reluctancy to interaction between groups). Based on the realization of the model, it is then argued that increased segregation between groups reduces social interactions, unless fully compensated for by increased within-group interactions (see p. 6).

I am afraid that I am not fully following the argument as this seems tautological to me: we assume that, following the social network analogy, egos are reluctant to interact with alters if they are outside of their group, and then find that for the case of segregation (i.e., structurally induced setting apart of one group from others), total interaction declines. We assume egos to prefer to interact within their group and compensate the lack of contact with other groups by having a higher rate of intra-group interactions, and then find that the loss of interaction due to segregation can be compensated if egos interact more with alters in their group (which they already do given the assumed homophily). From my grasp of the research, the only case in which this reasoning would not appear circular is if interactions had some variability in their type, duration, and intensity, or if there was an upper limit on the number of interactions. However, from my understanding, interactions are treated as uniform, and individuals do not have a “budget for interactions” (see p 5.).

In short, I am perplexed by this paper. I am looking forward to Thursday's workshop in hopes of better understanding the model, its assumptions, and implications.

**Comment by yuzhouw313** [comment]
Hello Dr. Berman,
Thank you for sharing your research with us. I am deeply intrigued by the relation between the trend of residential/neighborhood segregation and economic development in consideration of the Urban Scaling Theory. 

I am not sure if I understand the concept of the representation of the outcome homophily and heterophobia, as you described it as not indicating personal choice but macro-level network segregation (see page 5). However, from a perspective from someone who is not familiar with urban environment related theories, I generally consider the individual preference the foundation of the aggregated level of segregation, and I am not sure I understand the process of calculating these two indicators (hg^hom and hg^het). Can you please elaborate on these two variables/indicators? And how can you build the aggregation form from individual segments?

Thank you again and look forward meeting you at Thursday's workshop for more information on this topic.

**Comment by shaangao** [comment]
The paper did a good job of describing the correlation between residential segregation and economic outcomes, especially in terms of individual income. I guess I would be interested in knowing more about the mechanisms behind this phenomenon. The paper briefly mentioned that it might have something to do with labor opportunities -- what might be the causal pathway from residential segregation to labor opportunities? E.g., is it because people's social networks and information sources (e.g., about opportunities) are limited to those living near them? Because of the different educational opportunities and crime rates between neighborhoods leading to different qualifications for jobs?

Related, there is the question of whether it is a causal relationship from residential segregation per se to economic outcomes. Could residential segregation in fact be a proxy for other factors in society, such as the overall level of discrimination, that impact both residential segregation and economic outcomes? 

On the other hand, for explanations such as job opportunity/education/etc. differences caused by segregation to be valid, it seems that we need to assume that the segregated groups differ significantly in these factors that are substantial for income/economy in the first place, or at least in qualities that lead to disparities in these factors. I think one of the comments above also mentioned this circularity. I'm interested in knowing more about follow-up research that could potentially tease apart the impact of segregation per se and the mixed effect of segregation + existing historical disparities in SES, education, social status, etc. that are reinforced by segregation.

**Comment by lbitsiko** [comment]
Dear Prof. Berman,

Many thanks for presenting your work. I found the research on homophily/heterophobia adjustments in urban scaling theory particularly interesting.
It seems convincing that instead of assuming homogenous interactions between city residents, that we create numerous groups interacting with each other with some probability chosen such that they exhibit homophily/heterophobia. Although I do not fully understand how averaged interactions are connected to economic outcomes as well as how these modifications to urban scaling laws (and their empirical effects) add to our understanding of mechanisms producing inequality, I would like to highlight another point. I could not but think of the "Effects of Racial Segregation on Economic Productivity in U.S. Cities" paper in an environmental neuroscience context. Frankly, I simply could not connect it to the intellectually stimulating environmental neuroscience discussion of the two optional readings.
With all that said, really looking forward to your Thursday presentation.

Kind regards,
Loizos

**Comment by zihua-uc** [comment]
Hi Dr. Berman,

I wonder how much of the racial segregation is due to taste-based racial biases and how much is based on statistical discrimination. Some people might avoid certain neighborhoods because they are unsafe, but that could be correlated with the racial composition of the neighborhood.
Also. what are the policy implications - how could the government "encourage" less racial segregation to increase economic output?

**Comment by beilrz** [comment]
Hello professor,

Thank you very much for the paper.  A question I have after reading the paper is it is practical to establish casual relationship between environment and neural expression. I read that environmental neuroscience focus on the environmental effect on a science, but is the causal relationship possible? The causal vs correlational relationship is a question we encounter a lot in social science research.

**Comment by y8script** [comment]
Hello Prof. Berman, 
Thank you for sharing this work! For an extension of this model, I'm thinking about the possibility of assigning different homophily/heterophobia values to different groups in the same city, which may account for the phenomenon that each group may have its idiosyncratic tendencies for in-group and out-group interaction. Moreover, groups may have distinct average socioeconomic statuses, which determine the magnitude of contribution to economic outputs and discount the importance of their actions. Do you think this could be a potentially interesting mechanism to model on?

**Comment by ShiyangLai** [comment]
Dear Prof. Berman,

Thank you for sharing your work. My question is about the examination of the effects of homophily/heterophobia on the economic outputs of cities. In the paper "Effects of Racial Segregation on Economic Productivity in U.S. Cities", you showed a negative correlation between the constructed heterophobia measure and income scaling deviation. Does this necessarily mean that heterophobia brings negative impacts on economic outputs of cities? Or say, is there any reciprocal causality issue between homophily/heterophobia and economic development?

**Comment by bermanm** [comment]
> I am afraid that I do not fully grasp the assumptions and logic of the model on homophily and heterophobia, and/or parts of it may partly proceed in circular fashion.
> 
> If my understanding of the _Effects of Racial Segregation on Economic Productivity in U.S. Cities_ is correct, then the coefficients for homophily and heterophobia capture both structural factors and (potential) agentic choices that result in network segregation (i.e., outcome homophily and heterophobia, as described on p. 5). Individuals do not interact with each other equally likely, but belong to different groups, each with their own group-specific interaction rate. Individuals in each group are modelled as interacting preferentially with others of the same group, and with a lower probability with other groups. In case they lose contact with other groups, they may increase their rate of intra-group interactions (see p. 4). Put differently, the base assumption is some degree of homophily (preferential interaction within-group, be it structural and/or agentic) and heterophobia (reluctancy to interaction between groups). Based on the realization of the model, it is then argued that increased segregation between groups reduces social interactions, unless fully compensated for by increased within-group interactions (see p. 6).
> 
> I am afraid that I am not fully following the argument as this seems tautological to me: we assume that, following the social network analogy, egos are reluctant to interact with alters if they are outside of their group, and then find that for the case of segregation (i.e., structurally induced setting apart of one group from others), total interaction declines. We assume egos to prefer to interact within their group and compensate the lack of contact with other groups by having a higher rate of intra-group interactions, and then find that the loss of interaction due to segregation can be compensated if egos interact more with alters in their group (which they already do given the assumed homophily). From my grasp of the research, the only case in which this reasoning would not appear circular is if interactions had some variability in their type, duration, and intensity, or if there was an upper limit on the number of interactions. However, from my understanding, interactions are treated as uniform, and individuals do not have a “budget for interactions” (see p 5.).
> 
> In short, I am perplexed by this paper. I am looking forward to Thursday's workshop in hopes of better understanding the model, its assumptions, and implications.



> I am afraid that I do not fully grasp the assumptions and logic of the model on homophily and heterophobia, and/or parts of it may partly proceed in circular fashion.
> 
> If my understanding of the _Effects of Racial Segregation on Economic Productivity in U.S. Cities_ is correct, then the coefficients for homophily and heterophobia capture both structural factors and (potential) agentic choices that result in network segregation (i.e., outcome homophily and heterophobia, as described on p. 5). Individuals do not interact with each other equally likely, but belong to different groups, each with their own group-specific interaction rate. Individuals in each group are modelled as interacting preferentially with others of the same group, and with a lower probability with other groups. In case they lose contact with other groups, they may increase their rate of intra-group interactions (see p. 4). Put differently, the base assumption is some degree of homophily (preferential interaction within-group, be it structural and/or agentic) and heterophobia (reluctancy to interaction between groups). Based on the realization of the model, it is then argued that increased segregation between groups reduces social interactions, unless fully compensated for by increased within-group interactions (see p. 6).
> 
> I am afraid that I am not fully following the argument as this seems tautological to me: we assume that, following the social network analogy, egos are reluctant to interact with alters if they are outside of their group, and then find that for the case of segregation (i.e., structurally induced setting apart of one group from others), total interaction declines. We assume egos to prefer to interact within their group and compensate the lack of contact with other groups by having a higher rate of intra-group interactions, and then find that the loss of interaction due to segregation can be compensated if egos interact more with alters in their group (which they already do given the assumed homophily). From my grasp of the research, the only case in which this reasoning would not appear circular is if interactions had some variability in their type, duration, and intensity, or if there was an upper limit on the number of interactions. However, from my understanding, interactions are treated as uniform, and individuals do not have a “budget for interactions” (see p 5.).
> 
> In short, I am perplexed by this paper. I am looking forward to Thursday's workshop in hopes of better understanding the model, its assumptions, and implications.

Marc: Sorry that you were perplexed.  We cannot say anything agent preferences.  We are only observing demographic segregation and then calculating homophily and heterophobia from that (OLS regression; Equations 8&9 in the supplement).  We don't model an interaction budget and all agents have the same interaction cross section, a0.  Of course we could add complexity to the model and add a budget and have interaction cross sections vary for different people.  Also, there need not be any compensatory mechanism between homophily and heterophobia, but maybe I am not quite understanding what you are getting that there.  Hopefully, the talk tomorrow will help to clarify some things, but if not, I'm happy to chat more.

**Comment by bermanm** [comment]
> Hello Dr. Berman, Thank you for sharing your research with us. I am deeply intrigued by the relation between the trend of residential/neighborhood segregation and economic development in consideration of the Urban Scaling Theory.
> 
> I am not sure if I understand the concept of the representation of the outcome homophily and heterophobia, as you described it as not indicating personal choice but macro-level network segregation (see page 5). However, from a perspective from someone who is not familiar with urban environment related theories, I generally consider the individual preference the foundation of the aggregated level of segregation, and I am not sure I understand the process of calculating these two indicators (hg^hom and hg^het). Can you please elaborate on these two variables/indicators? And how can you build the aggregation form from individual segments?
> 
> Thank you again and look forward meeting you at Thursday's workshop for more information on this topic.

Marc:  Right, it is not personal choice and it all based on neighborhood segregation.  Please see supplemental equations 8 and 9.  I will discuss this more tomorrow.  Sylvan and others had some similar questions.

**Comment by iefis** [comment]
Hi Dr Berman,
Thank you in advance for presenting your work. I am curious about the empirical observation that the degree to which residential segragation is associated with economic productivity increased between 2010s while racial segregation in US cities was relatively constant. What are some latent factors that may explain to this observation? Thanks!

**Comment by bermanm** [comment]
> The paper did a good job of describing the correlation between residential segregation and economic outcomes, especially in terms of individual income. I guess I would be interested in knowing more about the mechanisms behind this phenomenon. The paper briefly mentioned that it might have something to do with labor opportunities -- what might be the causal pathway from residential segregation to labor opportunities? E.g., is it because people's social networks and information sources (e.g., about opportunities) are limited to those living near them? Because of the different educational opportunities and crime rates between neighborhoods leading to different qualifications for jobs?
> 
> Related, there is the question of whether it is a causal relationship from residential segregation per se to economic outcomes. Could residential segregation in fact be a proxy for other factors in society, such as the overall level of discrimination, that impact both residential segregation and economic outcomes?
> 
> On the other hand, for explanations such as job opportunity/education/etc. differences caused by segregation to be valid, it seems that we need to assume that the segregated groups differ significantly in these factors that are substantial for income/economy in the first place, or at least in qualities that lead to disparities in these factors. I think one of the comments above also mentioned this circularity. I'm interested in knowing more about follow-up research that could potentially tease apart the impact of segregation per se and the mixed effect of segregation + existing historical disparities in SES, education, social status, etc. that are reinforced by segregation.

Marc:  Right, we can't get at causality here per se.  In fact, I believe it is even more complicated because there is likely circular causality, e.g., biases might lead to segregation, which might lead to more biases, etc.  The reverse could also be true.  I think it is very important to think about how we might test for causality.  It is not trivial.  With that said, I still think we can learn a lot, even if we cannot make strong causal claims.  If a city is not taking advantage of its full social network, that is bad, and may cause it to underperform economically.

**Comment by bermanm** [comment]
> Dear Prof. Berman,
> 
> Many thanks for presenting your work. I found the research on homophily/heterophobia adjustments in urban scaling theory particularly interesting. It seems convincing that instead of assuming homogenous interactions between city residents, that we create numerous groups interacting with each other with some probability chosen such that they exhibit homophily/heterophobia. Although I do not fully understand how averaged interactions are connected to economic outcomes as well as how these modifications to urban scaling laws (and their empirical effects) add to our understanding of mechanisms producing inequality, I would like to highlight another point. I could not but think of the "Effects of Racial Segregation on Economic Productivity in U.S. Cities" paper in an environmental neuroscience context. Frankly, I simply could not connect it to the intellectually stimulating environmental neuroscience discussion of the two optional readings. With all that said, really looking forward to your Thursday presentation.
> 
> Kind regards, Loizos

Marc:  Thanks, Loizos.  Hopefully it is not an error in judgment on my part to present a few different topics tomorrow that are under the umbrella of Environmental Neuroscience.  I will try to make the connections more clear tomorrow.  For a quick framing here, my lab focuses on how the physical and social environment affect brain and behavior.  Our goal would be to be able articulate these relationships in quantitative terms.  Admittedly, it is a broad research endeavor.  What I will try to show you tomorrow, is how we think of brains, brain-environment systems, and brain-brain-environment systems through of lens of hierarchical systems theory.  In that sense, it may have been better for me to assign a different optional reading, namely, Berman, Stier and Akcelik, 2019, American Psychologist.  I will talk about some of our neuroscience research, and natural environment psychology research as appetizers, and then will move into urban scaling theory and heterophobia/homophily.  We have not linked that work to the brain, yet, but there are some datasets that will allow us to.  We shall see if I am successful tomorrow, but it not, you know where to find me to ask questions for clarification.

**Comment by erweinstein** [comment]
Hi Professor Berman,

Considering these papers collectively as part of a larger research program, I'm excited to see more examples of future research studying the possible relationships between characteristics of the human/built environment and outcomes we care about in the social and behavioral sciences (like mental health and labor market opportunities).

Looking specifically at something you and your co-authors mentioned in "Effects of Racial Segregation on Economic Productivity in U.S. Cities":

> ...in 2020, the New Orleans, Louisiana Metropolitan Area had a median income of approximately $54,400. The urban scaling law predicts a median income of approximately $65,000 from city size alone, while accounting for segregation brings the prediction to approximately $56,500.


1) While I would not be surprised at all by the idea that racial segregation has harmed the economic development of New Orleans, doesn't that city's recent experience of suffering [one of the most economically damaging natural disasters](https://en.wikipedia.org/wiki/Effects_of_Hurricane_Katrina_in_New_Orleans#Economy) in US history make it a poor point of comparison for demonstrating how well your method of adjusting for segregation explains economic under-performance relative to city size?

2) What do the comparable calculations for Chicago and other major US cities look like? (Assuming you didn't already plan to include that in your slides tomorrow.) I think that would be very interesting to see, particularly since Chicago (while being a "success story" of recovering and rebuilding from a much-longer ago disaster, the Great Chicago Fire of 1871) is often described as being far more physically/spatially racially segregated than our peer cities.

**Comment by XTang685** [comment]
Dear Prof. Berman,

Thank you for sharing your research with us! I am not so familiar with urban environment topics and I am wondering how the impact of segregation could affect us. Looking forward to your presentation on Thursday!

**Comment by yutaili** [comment]
Hi Marc,

Thanks for sharing this paper from Andrew and other collaborators. My questions are as follows:

1. In the urban scaling theory part, I think you've made an important assumption that the economic outputs are directly proportional to the social interactions in general $(A^{het}, A^{hom})$. Intuitively this assumption sounds valid because we tend to believe that interactions will foster beneficial outcomes, but are there empirical evidence that support this assumption?

2. The analysis part also consider the explained variance of scaling deviation $R^2$ as a factor in addition to the average interaction $A^{het}$. Similarly, do you think the deviation of incomes (in addition to median income) between neighborhoods/cities might also be an interesting factor to look at? Do people have more interactions when the adjacent neighborhoods have similar income levels? Thanks.

**Comment by awaidyasin** [comment]
Hi Prof., while it is reasonable to believe that racial segregation (which I understand is assumed to be mostly exogenous in the paper) would create a smaller economic surplus, I can't see what an appropriate policy response would be in this situation. Govts cannot possibly control the demographic mixture of a neighborhood or mandate individuals to live in certain neighborhoods or interact with certain people.

**Comment by Ry-Wu** [comment]
Hi Professor Berman, 
Thank you for sharing with us an interesting research. I'm wondering how to disentangle the effects of segregation from other factors that may also affect economic productivity. Looking forward to your presentation on Thursday!

**Comment by linhui1020** [comment]
Prof. Berman,
Thanks for your research. It is a very interesting research projects - combining racial segregation, urban environment and the economic productivity. Is the simulation of interaction between different social groups considering multiple socioeconomic characteristics? Looking forward to the presentations on Thursday.

**Comment by WonjeYun** [comment]
Dear Dr.Berman

Thank you for providing an insight into how segregation within the city could have some relation with the economic output. The question I had was how would the individual cities look like according to your results. In the U.S, there are cities that are known to have diverse ethnicity, and some that are not. It would be interesting to compare the cities, and possibly uncover the myths of the 'diverse' and 'segregated' cities. Furthermore, I wonder if there was an additional dynamic analysis of the GDP and median income. I think if the analysis of individual cities has showed diverse results, the argument that economic output and segregation having relation might have been more interesting.

**Comment by zoeyjiao1104** [comment]
Hi Professor Berman,

Thank you for sharing your work with us. In terms of investigating the effects of homophily and heterophobia on the economic productivity, are there any important confounding factors that we should take into account? Looking forward to your presentation tomorrow!

**Comment by Hongkai040** [comment]
Hi Professor Berman,

Thank you for your team's work! I have to say linking racial segregation and economic productivity is really a natural and fantastic idea! Admitting the marvel of this idea, I have a question lingering in my mind: Is this connection real? (I guess that's what all fantastic ideas must go through)

My main concern is that economic productivity is most directly related to work happening in the working space while the data obtained to measure racial segregation is about our location after finishing one day's work. While the result shows an increased segregation is linked to decreased economic productivity, I am afraid that the model doesn't include the most important variable, racial segregation in the working space, which you were also aware of and mentioned in the discussion. 

Also from the perspective of racial segregation, I can have a totally different story that could lead to another conclusion: People prefer to interact with their kind(homophily) However, they have to work with people from other races, which makes them unhappy and inefficient. In order to compensate for the unhappiness during work, they decide to move to communities that have more members of their kind so that they can talk and hang out, but they still work inefficiently in working spaces. In this case, it's not the segregation in neighborhoods but actually the discordance in working spacing that leads to an economic productivity recession.  Maybe it's an unhappy truth and an inappropriate example, but maybe we can think about the ethnicity of most non-academic and non-administrative staff on our campus and where they may live after finishing one day's work. 

Looking forward to your presentation tomorrow and thank you for sharing your work!

**Comment by bningdling** [comment]
Hi Marc, 
Thanks for sharing your research with us! It surprises me that interactions among different racial groups can meaningfully impact their economic output. Sadly, as it was mentioned in the paper, segregation has been accelerating in the States. I wonder if you think "social comparison" might be another factor (other than peer influence and behavioral norms). I'm also curious about how to test those factors.

**Comment by javad-e** [comment]
Thank you so much for presenting this interesting paper, Prof. Berman!
I was wondering what your expectations are regarding the effects of other types of segregation. In particular, do you think income or wealth segregation would have similar effects on productivity as the observed effects from racial segregation?

**Comment by HongzhangXie** [comment]
Dear Professor Berman

Many thanks for sharing the interesting study with us! There are two points I am very interested. First, the empirical results find the effect of heterophobia on economic productivity is much more significant than homophily, even though they are highly correlated. The study makes a discussion from the perspective of racial segregation curtails and labour opportunities and says it could be operationalized in future work. I would like to learn more about the mechanism of these two indicators to economic productivity and the reasons for the difference in explained variance (13.7% vs 1.3%).

Second, the heterogeneous network structure in cities can significantly impact economic outputs. I think the economic outputs (income) may also impact the network structure. For example, individual income level is important in determining where and how to live. How could I consider the causal relationship between the network structure and economic outputs?

**Comment by YLHan97** [comment]
Hi Professor Berman,

Thanks for sharing your research with us. I have a question as follows:
In the article “Effects of Racial Segregation on Economic Productivity in U.S. Cities”, you have mentioned that empirical tests of the model show that increased residential racial heterophobia and segregation in U.S. cities are associated with reduced economic outputs and that the strength of this relationship increased throughout the 2010s. Since I’m really interested in the homophily and heterophobia applied in real world, but not so much familiar with it, would you please provide more real world examples in your relevant area?

**Comment by LynetteDang** [comment]
Hi Dr. Berman,

Thank you for sharing your work with us. Your work lies exactly in the intersection among urban science, neuroscience, and economics, it was so fascinating to see how all these different elements coming into play. Do you think ultimately there will be a solution for the increasing residential racial heterophobia segregation in the near future? I am looking forward to your presentation.

**Comment by koichionogi** [comment]
Thank you so much for introducing this exciting research!! I wondered how the finding of relationships in city size, segregation, diversity, depression rate, and economic outcomes could apply in other cities. Also, how would you like to extract the causal relationship of these variables?  Suppose the social theory with the model is accurate and accords with the statistics. What could be the next step to elucidate the mechanism between the size and characteristics of a community? Thank you so much

**Comment by lguo7** [comment]
Hi Dr. Berman,
Thank you so much for sharing your research. I'm curious if this model has any application to Chicago, in my opinion (both in real-life scenarios and in some of the literature), Chicago has more pronounced North-South segregation than other cities I've lived in, would you be able to give a general overview of the economic impact of segregation in Chicago tomorrow? Looking forward to your talk on Thursday!

**Comment by borlasekn** [comment]
Dr. Berman - thank you for sharing your research! I had a question regarding your operationalization of segregation. In the "Materials and Methods" section of the required paper for this week, you state: "The construction of measures of segregation based more directly on interactions, beyond the composition of residential neighborhoods". I was wondering if you were able to or would control for other factors related to the composition of residential neighborhoods, which might provide additional information about interactions. For example, I wonder about how either the (1) age of the neighborhood, or (2) age of residents in the neighborhood might contribute to residents' tendencies towards homophily or heterophobia? The former could be important when considering the history of spatial segregation. There is a history of both de facto and de jure spatial segregation. A neighborhood that is older may still be segregated as a result of previous de jure segregation, while a newer neighborhood (or even a neighborhood with changing demographics) might tell us more about people's interactive tendencies. The latter might contribute to how likely people are to interact with those different than themselves. I also wonder about other sociodemographic factors (gender composition, education, income, etc.) that might be both tied to a history of segregation in America and thus contribute to neighborhood concentration and impact trends in homophily. Thank you for the insight!

**Comment by tangn121** [comment]
Hello Dr. Berman,

Thank you for sharing your research on the association between racial segregation in U.S. cities and economic output. I am curious to know if this relationship holds true at the city level, and which factors should be taken into consideration when analyzing such correlations at the city level.

**Comment by Toushirow1** [comment]
Hi Dr. Berman, Thank you for sharing your profound research on cognitive and affective processing. I would like to know whether a similar research design pattern can be used to explore the effect of text/video content on cognition. Thank you!

**Comment by JunoWuu** [comment]
Hi Dr. Berman! Thank you for sharing your research. I read the paper on racial segregation. However, since your topic involves neurons and brain activity, I wonder how this topic can relate to neuroscience? Thank you!

**Comment by mintaow** [comment]
Hi Dr. Berman,

This research is very inspirational and the many profound questions raised got me thinking particularly about the association between individual economic status and racial segregation. I would be very curious to hear how we can exclude the confounding effects and test this hypothesis in a trustworthy way. Looking forward to the workshop tomorrow.

Thanks for you taking the time to share your work with us.

**Comment by hazelchc** [comment]
Hi Prof. Berman, Thank you so much for sharing this interesting paper with us! I’m curious about the policy implications of the paper. This paper shows that increased residential racial segregation is associated with reduced economic outputs, while other papers also demonstrate the negative impacts of racial segregation. How can this paper inform policymakers to formulate alleviating policy? Thank you!

**Comment by LuZhang0128** [comment]
Hi Dr. Berman,
Thank you for sharing such a fantastic paper with us. I am also a bit confused by the difference between individual preference and aggregated segregation. I also look forward to learning more about Environmental Neuroscience through tomorrow's workshop. Thank you!

**Comment by Yuxin-Ji** [comment]
Hi Dr. Berman,
Thank you for sharing the research. It is surprising to see how heterophobia can impact economic output. This finding is very meaningful as it suggests how segregation could harm the country as a whole. You mentioned in the paper that the level of racial segregation was relatively constant in US in the past ten years, I wonder if that is an overall statistics or hold true for majority of the cities? I am curious whether there are any changes or patterns in the past decade as people are constantly moving around. Thank you!

**Comment by yhchou0904** [comment]
Dear Prof. Berman,
Thank you for sharing your idea with us! We have seen many methods to capture the factors that influence economic outcomes, and using racial segregation might be one of the most interesting ones. While reading the paper, some questions come into my mind: when describing the settings, the study assumes that people are equally likely to interact or, say, are equally likely to interact within a group. Is it possible to introduce some heteroskedasticity in how likely people interact with each other, and if it is the case, would it affect how we measure the effect? Also, the research assumes that economic outputs are proportional to between and within interactions. Does the assumption come from observation or solid prior belief?

**Comment by jiehanL** [comment]
Hi Prof. Berman,

My questions is, what are the limitations and potential sources of bias in the study's model and methods for measuring the relationship between residential racial segregation, heterophobia and economic outputs in U.S cities?

**Comment by zhiyun0707** [comment]
Hi Professor Berman,
Thank you for presenting your work with us! My question is that when considering the interactions between city inhabitants of economic outcomes, you make the assumption that between and within group interactions are the same, but I think this may cause different economic outputs. How will the result change if we separate these two group?

**Comment by ChongyuFang** [comment]
Hi Prof. Berman, Thanks for sharing your research us. I wonder how the findings can link with behavioral economics and urban economics?

**Comment by yujing-syj** [comment]
Hi Dr. Berman, Thanks so much for sharing your interesting researches with us. Personally, I am very interested in you paper "Evidence and theory for lower rates of depression in larger US urban areas". The relationship between depression and large urban areas is a very popular topic. I really like your approach to get the results by using different kinds of data sets such as twitter, NSDUH and BRFSS. This gives us a comprehensive understanding of the robustness of the result. In terms of the result, the paper states that small cities actually perform worse as they bear the brunt of the social and economic burden of depressive disorders. I am wondering whether there are some studies are detecting or monitoring the change of the depression rate with the development of the specific city? It would be interesting to see the dynamic change and their vertical relationship.

**Comment by Coco-Jiachen-Yu** [comment]
Hello Professor Berman,

Thank you so much for presenting your study with us. I found it interesting that racial segregation is more strongly associated with individual economic output than overall economic output. What do you think are possible reasons behind this? What study design can you think of to test your hypothesis?

**Comment by yjhuang99** [comment]
Hi Prof. Berman, I enjoy reading your papers, which nicely combine theory and empirical evidence. I am especially interested in your paper "Effects of Racial Segregation on Economic Productivity in U.S. Cities" - I actually wonder if the understanding of homophily and heterophobia in social networks and their corresponding effects on the economy has some policy implications. What can we do after observing these patterns? Even if we quantify the adverse effects, it seems hard to force individuals and communities to change... I look forward to your interpretations tomorrow.

**Comment by fiofiofiona** [comment]
Hi Dr. Berman, thank you for sharing your work in our workshop. In the paper "Effects of Racial Segregation on Economic Productivity in U.S. Cities", the analysis was conducted on the city level in the US; do you think a similar pattern would be found if using countries as units on a global scale study?

**Comment by zyang39** [comment]
Thank you for sharing your works, professor. The argument is really new to me and inspiring. I am wondering if it is doable to study this problem across countries.

**Comment by hshi420** [comment]
Considering factors like city size as first-order context, and then factors like the cultural background of people inside segregated groups can be second-order context. How would second-order context affect the human cognitive process? Is it possible to measure this kind of second-order context on a large scale? Thank you very much!

**Comment by Peihan12** [comment]
Hi Professor Berman,
Thank you for sharing with your work. I'm curious how to separate the impacts of segregation from other factors and if there are any policy implications associated with our growing understanding of homophily and heterophobia in social networks and their related consequences on the economy.

**Comment by ddlee19** [comment]
Hi Prof Berman,

I'm curious about learning more about homophily! Excited to hear your talk tomorrow.

**Comment by Emily-fyeh** [comment]
Hi Prof. Berman,
Thank you for sharing your work. The construct of this paper is relatively new to me since I've never thought of linking individual income with geographical racial separation before. Given the US Census Data has information on race and ethnicity--where ethnicity simply implies the identity of Hispanic and non-Hispanic. I would like to know if you would expect different outcomes when considering ethnicity.

**Comment by jinyz1220** [comment]
Thank you, Dr. Berman, for sharing your work. I would like to know what roles conformity and social comparison theory play in the strong association between individual economic output and regional segregation. In addition, how does the awareness of the correlation between the regional economy and homophily affect people's life choices and political stances?

**Comment by shenyc16** [comment]
Hi Prof. Berman,
Thank you for sharing this inspiring work with us. It is really interesting to think about how racial heterophobia and segregation can influence economic productivity. Do you think there are other confounding factors that may impact or even overturn the current results? What are they and how you may deal with them?

**Comment by BaotongZh** [comment]
Hi Prof. Berman, 
Thank you for sharing your work. I was just wondering whether the personal preferences for heterophobia are considered as exogenous shock? Are their preferences meant to make the economic output worse off spontaneously. if so what could we do to prevent that.

**Comment by xin2006** [comment]
Hi Dr. Berman
Thank you for sharing the work. Homophily is a topic I am interested in in the social network analysis. There is a tendency that members in a social group would connect with individuals similar with them in some attributes. On the other hand, much connection among people might result in the similarity among them. How do you think about the mutual effect in terms of Homophily?

**Comment by nijingwen** [comment]
Hi, Berman
Thank you for sharing the work. You mentioned the increasing number of parks can reduce crimes. Beside emotional help, do you think there are more economic reasons? because the place has parks should have better regulation and richer

**Comment by ZHE-ZHANG-0213** [comment]
Hi Professor Berman,
Thank you for sharing with us an interesting research.  I'm wondering how to disentangle the effects of segregation from other factors that may also affect economic productivity. And I can't see what an appropriate policy response would be in this situation.  Do you have any suggestions in this regard? Excited to hear your talk tomorrow.

**Comment by YutaoHeOVO** [comment]
Hi Professor Berman,

Thank you for enlightening us with an interesting research paper. Your research reminds me of a paper _The Wrong Side(s) of the Tracks: The Causal Effects of Racial Segregation on Urban Poverty and Inequality_. The effect of segregation on economic growth is notorious for its endogeneity problems: for instance, there can be unobserved economic, political or other attributes that make people make migration decisions. Moreover, there can be simultaneous causality involving the whole process: while segregation slows down the economic growth, the relative worse characteristics in population can also trigger people to make decisions in migration. (After all, migration is something by choice.) And this can worsen the problem of segregation. And I guess the causal relationship is like:

                                             Homophily/Heterophobia > Neighborhood preference > Segregation

And homophily and heterophobia, though endogenous is relatively fixed which cannot be affected by segregation? And I wonder if you could explain further how to establish the causal relationship and alleviate the endogeneity problem at the same time. Moreover, I am also curious of an individual's decision process: suppose that agent's degree of homophily/heterophobia can affect his preference on how segregate a city (or more micro-level, a neighborhood) is, then what about other attributes? Can they also trigger people's decisions and then cause segregation issue? (Something like the omitted variable bias argument proposed by Cutler and Glaeser in their paper _Are Ghettos Good or Bad?_ (And at least by Tiebout sorting, the other tributes like public goods indeed can affect individuals' decision on places to live I think?)

Thank you so much if you can answer my questions.

Best,
Yutao

**Comment by hsinkengling** [comment]
Hi Prof. Berman

Thanks for sharing your work on segregation and economic outputs. Under the assumption of homophily and heterophobia, wouldn't it make sense that areas with higher segregation have more interaction, since people are less afraid to interact with neighbors, compared with areas with low segregation, where people might meet less people-of-their-own-kind?

**Comment by JerryCG** [comment]
Hi Prof. Berman,

The racial problem has always been a social issue throughout human history. Do you think homophily (or heterophobia) always harms the economics performance? Or maybe some level of homophily (or heterophobia) can be actually beneficial to the society? Is an perfectly diversifed and united society necessarily acheiving the best economic outcome?

Best,
Jerry Cheng

**Comment by bhavyapan** [comment]
Thank you for presenting your research, Dr. Berman. My apologies for the late comment.

What are some of the ways in which research design can be leveraged to study the long-term economic implications of the main research question, and potentially across different contexts - say geographically or culturally?

Best regards, 
Bhavya

**Comment by helyap** [comment]
Hi Professor Berman, 

Thank you for once again sharing your work with us. Considering the associative results from the study of homophilous and heterophobia effects on economic productivity, I'm curious about potential moderators/mediators that may interrupt those effects. 

Thank you,
Helen

**Comment by xinyi030** [comment]
Hi Dear Prof. Berman,

Thanks again for sharing your work. Just want to check in with you that I previously wrote a comment on this paper on another issue page which is currently closed. Here's a screenshot of my comment. Thanks! 

![140391678721944_ pic](https://user-images.githubusercontent.com/112563982/224760446-e5574000-4f7e-4d1d-b935-414e5ffd5d1c.jpg)
